src/modeler/AutoRest.Swagger/Validation/PutGetPatchResponseValidation.cs

```diff
@@ -39,51 +39,27 @@ public class PutGetPatchResponseValidation : TypedRule<Dictionary<string, Dictio
         /// </summary>
         public override Category Severity => Category.Error;
 
-        /// <summary>
-        /// Validates if the response of Put/Get/Patch are same.
-        /// </summary>
-        /// <param name="paths">paths to validate</param>
-        /// <param name="formatParameters">The noun to be put in the failure message</param>
-        /// <returns>true if the response is same for Put/Get/Patch responses.false otherwise.</returns>
-        public override bool IsValid(Dictionary<string, Dictionary<string, Operation>> paths, RuleContext context, out object[] formatParameters)
+        private static readonly IEnumerable<string> OpList = new List<string>() { "put", "get", "patch" };
```

Comment:
is the path from using "IsValid" ever correct? I wonder how useful that method is and/or if it should be updated to receive a path.